mmcv/runner/hooks/lr_updater.py

```diff
@@ -65,10 +67,10 @@ def _set_lr(self, runner, lr_groups):
                                        lr_groups):
                 param_group['lr'] = lr
 
-    def get_lr(self, runner, base_lr):
+    def get_lr(self, runner: 'runner.BaseRunner', base_lr: float):
```

Comment:
We don't need to add type hints for `runner` in hooks. If we use relative import, it will cause circular import. If we use absolute import like this, we need to add an extra `type: ignore`